Assets/MixedRealityToolkit.Tools/ReserializeAssetsUtility/ReserializeAssetsUtility.cs

```diff
@@ -29,6 +30,23 @@ private static void ReserializeMaterials()
             AssetDatabase.ForceReserializeAssets(array);
         }
 
+        [MenuItem("Assets/Mixed Reality Toolkit/Reserialize Selection")]
+        public static void ReserializeSelection()
```

Comment:
@gilbdev could you please add comments for this public method?

Reply:
Ah just realized that I merged a bit early. No worries on these comments, if you have time and could please add these in another PR (you can reference the same issue) that would be great.

Reply:
Sadly i commit this after on but forgot to push 😝